diff --git a/src/ftpd.c b/src/ftpd.c
index 53f4de4..003734d 100644
--- a/src/ftpd.c
+++ b/src/ftpd.c
@@ -1069,8 +1069,6 @@ void dobanner(const int type)
 
 #endif
 
-#ifndef MINIMAL
-
 int modernformat(const char *file, char *target, size_t target_size,
                  const char * const prefix)
 {
@@ -1217,8 +1215,6 @@ void doallo(const off_t size)
     }
 }
 
-#endif
-
 void dositetime(void)
 {
     char tmp[64];

