diff --git a/src/ftpd.c b/src/ftpd.c
index 95bf1e5..5c0723e 100644
--- a/src/ftpd.c
+++ b/src/ftpd.c
@@ -1087,8 +1087,6 @@ void dobanner(const int type)
 
 #endif
 
-#ifndef MINIMAL
-
 int modernformat(const char *file, char *target, size_t target_size,
                  const char * const prefix)
 {
@@ -1235,8 +1233,6 @@ void doallo(const off_t size)
     }
 }
 
-#endif
-
 void dositetime(void)
 {
     char tmp[64];
